GUI/Main.Designer.cs

```diff
@@ -263,16 +262,17 @@ private void InitializeComponent()
             this.aboutToolStripMenuItem.Text = "About";
             this.aboutToolStripMenuItem.Click += new System.EventHandler(this.aboutToolStripMenuItem_Click);
             // 
+            //
             // statusStrip1
-            // 
-            this.statusStrip1.ImageScalingSize = new System.Drawing.Size(24, 24);
-            this.statusStrip1.Location = new System.Drawing.Point(0, 1016);
```

Comment:
Leave `Location` in

Reply:
Oh, the IDE probably needs that stuff for the graphical editor, doesn't it? OK...

Reply:
Done